gradle.properties

```diff
@@ -48,11 +50,11 @@ junit.version=4.12
 
 # Maven Repository (i.e. Sonatype Nexus Repository Manager) Configuration --------------------------
 
-NEXUS_USERNAME=
-NEXUS_PASSWORD=
+NEXUS_USERNAME=admin
+NEXUS_PASSWORD=admin123
```

Comment:
No that has to be there. well I can null it out then add it via the ci
scripts, but the new ci tests actually downloads nexus, starts it,
publishes to it, then confirms everything that's supposed to be there, is
(and is signed)

On Mon, Sep 5, 2016 at 4:30 PM, Chris Doyle <notifications@github.com>
wrote:

> In gradle.properties
> <https://github.com/chrisdoyle/gradle-fury/pull/66#discussion_r77554482>:
>
> > @@ -48,11 +50,11 @@ junit.version=4.12
> >
> >  # Maven Repository (i.e. Sonatype Nexus Repository Manager) Configuration --------------------------
> >
> > -NEXUS_USERNAME=
> > -NEXUS_PASSWORD=
> > +NEXUS_USERNAME=admin
> > +NEXUS_PASSWORD=admin123
>
> Oops :)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/chrisdoyle/gradle-fury/pull/66/files/c24d7742f34ddd5e22ec5f0e10ded9667c884691#r77554482>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB4kGT6X589YYJoTKNR9jCRthYbt5FUIks5qnHvegaJpZM4J1Sd9>
> .
>